scripts/commonfarms-cli/commonfarms-cli

```diff
@@ -13,18 +13,23 @@ def argument_parser():
     parser = ArgumentParser(prog='commonfarms-cli', description='Command line tool for management of Common Farms')
     parser.add_argument('--chain', metavar='URL', default='local', help='WebSocker URL of the chain (possible shortcuts: mainnet, testnet, local)')
     parser.add_argument('--phrase', metavar='SEED', help='seed phrase of the farm admin account (if not supplied, an interactive prompt will ask for it)')
-    parser.add_argument('--metadata', metavar='PATH', default='../../artifacts/farm_contract.json', help='path to farm contract metadata file')
+    parser.add_argument('--farm-metadata', metavar='PATH', default='../../artifacts/farm_contract.json', help='path to farm contract metadata file')
+    parser.add_argument('--psp22-metadata', metavar='PATH', default='../../artifacts/psp22.json', help='path to PSP22 contract metadata file')
```

Comment:
We could consider dropping these two arguments and fetching them from `artifacts`

Reply:
I can imagine someone just "copying" the file w/o arficats. Let's leave as-is, defaults are good.